get-stats.sh

```diff
@@ -4,19 +4,9 @@ set -euo pipefail
 # Note that it is also possible to get stats from previous months via
 # /api/stats?year=2022&month=1
 
-curl \
+exec curl \
     --user-agent "cargo-quickinstall build pipeline (alsuren@gmail.com)" \
     --silent \
     --show-error \
     -XGET \
-    "https://warehouse-clerk-tmp.vercel.app/api/stats" | (
-    # Slight hack: if TARGET_ARCH is specified then just print crate names one per line.
-    # Otherwise print all counts as json.
-    if [[ "${TARGET_ARCH:-}" != "" ]]; then
-        jq -r 'keys | .[]' |
-            (grep -F "${TARGET_ARCH:-}" || true) |
-            sed 's:/.*::' || (echo "Invalid json from stats server. Probably rate limit error. Skipping for now." >&2)
```

Comment:
these `|| true` and `|| echo` things are there because the stats server might have hit its rate limit or the grep might return 0 lines (in which case it "fails" and -o pipefail causes that failure to propagate).

Reply:
Done!